src/Compilers/VisualBasic/Test/Syntax/Parser/ParserRegressionTests.vb

```diff
@@ -934,4 +934,29 @@ End Enum
         Assert.Equal(Syntax.InternalSyntax.Scanner.BadTokenCountLimit, tree.GetDiagnostics().Where(Function(d) d.Code = ERRID.ERR_IllegalChar).Count())
     End Sub
 
+    <Fact, WorkItem(48587, "https://github.com/dotnet/roslyn/issues/48587")>
+    Public Sub ParseTrailingTextAfterPropertyWithParentheses()
+        Dim compilation = CompilationUtils.CreateCompilationWithMscorlib40(
+<compilation name="ParseTrailingTextAfterPropertyWithParentheses">
+    <file name="a.b">
+Class C
+    Public ReadOnly Property NumberOfResult1() String Integer JohnDoe WwwIIWww Wow
+    Public ReadOnly Property NumberOfResult2() Some unexpected tokens As Integer
+
+    Public ReadOnly Property NumberOfResult3() ' With comment - no errors.
```

Comment:
Can you please also add a version with bad text in between the parens and the comment/line continuation below?